doc/odml_ontology/root-ontology.ttl

```diff
@@ -0,0 +1,330 @@
+                                                   rdfs:domain <https://g-node.org/projects/odml-rdf#Document> ,
+                                                               <https://g-node.org/projects/odml-rdf#Section> ;
+                                                   rdfs:range <https://g-node.org/projects/odml-rdf#Terminology> ;
+                                                   rdfs:label "hasTerminology"^^xsd:string .
+
+
+#################################################################
+#    Data properties
+#################################################################
+
+###  https://g-node.org/projects/odml-rdf#author
+<https://g-node.org/projects/odml-rdf#author> rdf:type owl:DatatypeProperty ;
+                                              rdfs:domain <https://g-node.org/projects/odml-rdf#Document> ;
+                                              rdfs:range xsd:string ;
+                                              rdfs:label "author" .
+
+
+###  https://g-node.org/projects/odml-rdf#date
+<https://g-node.org/projects/odml-rdf#date> rdf:type owl:DatatypeProperty ;
+                                            rdfs:domain <https://g-node.org/projects/odml-rdf#Document> ;
+                                            rdfs:range xsd:date ;
+                                            rdfs:label "date" .
+
+
+###  https://g-node.org/projects/odml-rdf#definition
+<https://g-node.org/projects/odml-rdf#definition> rdf:type owl:DatatypeProperty ;
+                                                  rdfs:domain <https://g-node.org/projects/odml-rdf#Property> ,
+                                                              <https://g-node.org/projects/odml-rdf#Section> ;
+                                                  rdfs:range xsd:string ;
+                                                  rdfs:label "definition" .
+
+
+###  https://g-node.org/projects/odml-rdf#doc-version
+<https://g-node.org/projects/odml-rdf#doc-version> rdf:type owl:DatatypeProperty ;
+                                                   rdfs:domain <https://g-node.org/projects/odml-rdf#Document> ;
+                                                   rdfs:range xsd:float ;
+                                                   rdfs:label "doc-version" .
+
+
+###  https://g-node.org/projects/odml-rdf#dtype
+<https://g-node.org/projects/odml-rdf#dtype> rdf:type owl:DatatypeProperty ;
+                                             rdfs:domain <https://g-node.org/projects/odml-rdf#Property> ;
+                                             rdfs:range xsd:string ;
+                                             rdfs:label "dtype" .
+
+
+###  https://g-node.org/projects/odml-rdf#hasResource
+<https://g-node.org/projects/odml-rdf#hasResource> rdf:type owl:DatatypeProperty ;
+                                                   rdfs:domain <https://g-node.org/projects/odml-rdf#Terminology> ;
+                                                   rdfs:range rdf:XMLLiteral ;
```

Comment:
I'll add `odml:hasTerminology`. What about the name of the property pointing to an external resource? Should we leave `hasResource`?